lib/littlebox.css

```diff
@@ -3261,26 +3261,26 @@
 .lb-flag-france {
```

Comment:
Thanks for catching that! There were a few leftovers in there. Getting rid of them now.